File: src/test/java/android/test/regression/CompleteRegressionAndroidSuite.java
package android.test.regression;

import android.test.sample.SampleAndroidSuite;
import org.junit.experimental.categories.Categories;
import org.junit.runner.RunWith;

import static org.junit.runners.Suite.SuiteClasses;

@SuiteClasses({
    SampleAndroidSuite.class
})
@RunWith(Categories.class)
public final class CompleteRegressionAndroidSuite {
}

File: src/test/java/android/test/regression/CompleteRegressionAndroidSuite.kt
package android.test.regression

import android.test.helper.SetPlatformAndroidHelper
import common.test.regression.CompleteRegressionSuite
import org.junit.experimental.categories.Categories
import org.junit.runner.RunWith
import org.junit.runners.Suite.SuiteClasses

@SuiteClasses(
    SetPlatformAndroidHelper::class,
    CompleteRegressionSuite::class
)
@RunWith(Categories::class)
class CompleteRegressionAndroidSuite